jenkinsfiles/Jenkinsfile.grobid

```diff
@@ -1,9 +1,11 @@
 elifePipeline {
     node('containers-jenkins-plugin') {
         def commit
+        def revision = params.revision ? params.revision : 'catchall'
+        def tag = params.revision ? params.revision : 'latest'
 
         stage "Checkout", {
-            sh "rm -rf grobid && git clone -b catchall git@github.com:elifesciences/grobid.git"
+            sh "rm -rf grobid && git clone -b ${revision} git@github.com:elifesciences/grobid.git"
```

Comment:
maybe if we change it to something like this it would support commits:

```
sh "rm -rf grobid && git clone -b catchall git@github.com:elifesciences/grobid.git && cd grobid && git checkout ${revision}"
```

For the main case that should be as efficient. For other commits it might have to do more work.

Reply:
Ah, didn't test. Open a PR and I'll approve?